src/app/components/NewsList/NewsList.module.scss

```diff
@@ -0,0 +1,16 @@
+@use '@/styles/variables.scss' as *;
+@use '@/styles/mixins.scss' as *;
+
+.newsList {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
```

Comment:
It’s better to use a mobile first approach to improve performance on mobile devices, so please rewrite your media queries respectively and change styles to  
```
.newsList {
  display: grid;
  grid-template-columns:  1fr;
  gap: $spacing-l;

  @include from-tablet {
    grid-template-columns: repeat(2, 1fr);
  }

  @include from-desktop {
    grid-template-columns: repeat(3, 1fr);
  }
}
```